rooms/windyTylne.cpp

```diff
@@ -1,24 +1,215 @@
+        << "Widzisz porwane kable bez izolacji, które czasami się stykając"
+        << "tworzą efektowne zwarcia" << std::endl
+        << "Co robisz?" << std::endl
+        << "1. Naprawiam dalej" << std::endl
+        << "2. Odpuszczam i przykręcam płytkę z powrotem" << std::endl;
+
+    input_prompt(wybor);
+    if (wybor[0] == '1') {
+      std::cout << "Not implemented yet" << std::endl;
+      /*
+      std::cout << "Po chwili analizowania kabli stwierdzasz, że jak je
+      poprzesuwasz i odpowiednio połączysz"
+                << " to przyciski powinny zacząć działać" << std::endl;
+
+      bool kable_poprawnie = false;
+      bool ruch = false;
+
+      char kable [minigame_row][minigame_col+2] {};
+      short offset [minigame_col] {};
+
+      while(kable_poprawnie) {
+          while(!ruch) {
+              clear_screen();
+              draw_minigame(kable, offset);
+          }
+
+      }
+      */
+      return false;
+    }
+    return false;
   }
-  std::cout << "Czy chcesz iść do windy na parterze?" << std::endl;
-  if (yes()) {
-    return WINDA_PARTER;
+    std::cout << "Niestety, ale nie masz potrzebnego narzędzia do tego, wróć "
+                 "później jak je zdobędziesz"
+              << std::endl;
+    return false;
+}
+
+Room wybor_dzialania(GameData& gameData, int8_t i_c) {
+  std::string wybor;
+  while (true) {
+    std::cout << "Co robisz?" << std::endl;
+    wait_ms(200);
+    std::cout << (gameData.winda_repaired
+                      ? "1. Korzystam z windy i zjeżdżam na parter"
+                      : "1. Podchodzę do windy, by zobaczyć czemu nie działa")
+              << std::endl
+              << "2. Idę do windy z przodu budynku" << std::endl
+              << "3. Idę do biblioteki" << std::endl
+              << "4. Idę do garażu (i robię przyps)" << std::endl
+              << "5. Wracam na korytarz" << std::endl;
+
+    if (!gameData.zaionc) {
+      input_prompt(wybor);
+      if (wybor[0] == '1') {
+        if (!gameData.winda_repaired) {
+          std::cout << "Widzisz, migotające co jakiś czas, podświetlenie "
+                       "przycisku przywołania windy o jasnoniebieskim kolorze"
+                    << std::endl
+                    << "Po przyjrzeniu się zauważasz, że w środku musi być "
+                       "jakieś zwarcie"
+                    << std::endl
+                    << std::endl
+                    << "Co robisz?" << std::endl
+                    << "1. Próbuję naprawić" << std::endl
+                    << "2. Mam to w głęboko i próbuję użyć windy" << std::endl;
+
+          do {
+            input_prompt(wybor);
+            if (wybor[0] == '1') {
+              std::cout << "Not implemented yet" << std::endl;
+              return WINDY_TYLNE;
+              // if(lift_repairing_game(gameData)) gameData.winda_repaired =
+              // true;
+            }
+            if (wybor[0] == '2') {
+              if (i_c == 8) {
```

Comment:
Co to za zmienna?

Reply:
Masz na myśli i_c?

Reply:
tak, `i_c` - jej nazwa kompletnie nic mi nie mówi.